accessibility-oscilloscope/configs/rpi4_defconfig

```diff
@@ -10,7 +12,8 @@ BR2_ROOTFS_POST_SCRIPT_ARGS="--add-miniuart-bt-overlay"
 BR2_LINUX_KERNEL=y
 BR2_LINUX_KERNEL_CUSTOM_TARBALL=y
 BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="$(call github,raspberrypi,linux,967d45b29ca2902f031b867809d72e3b3d623e7a)/linux-967d45b29ca2902f031b867809d72e3b3d623e7a.tar.gz"
-BR2_LINUX_KERNEL_DEFCONFIG="bcm2711"
+BR2_LINUX_KERNEL_USE_CUSTOM_CONFIG=y
+BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="../accessibility-oscilloscope/configs/rpi4_linux_defconfig"
```

Comment:
is this kernel config based on the stock  `bcm2711` kconfig then? as in, we only add stuff, and this won't regress anything?

Reply:
yes it's based on what I had locally modified before, should have been just our changes on top of `bcm2711`.
bit of bloat possibly with all the modules & things we don't need but I didn't want to touch it